README.rst

```diff
@@ -34,11 +34,11 @@ generation to fail, or result in major rendering defects relative to what you
 intend.
 
 
-Browse the ReadTheDocs preview
+Browse the "Read the Docs" preview
 ------------------------------
```

Comment:
```suggestion
Browse the Read the Docs preview
--------------------------------
```

Reply:
Thanks for the good catch regarding the dashes, @hugovk!
I'm not sure about quotes. I've added them since `the Read the Docs` looks oddly overwhelmed with `the` and quotes make it looks a bit better in my opinion.
Maybe `Browse a Read the Docs preview` would be acceptable? 🤔

Reply:
Okay, let's keep the quotes. Please could you adjust the underline to match the width?

Reply:
Sure! I've updated changes.